app/src/main/java/com/univoice/feature/signup/DepartmentInputActivity.kt

```diff
@@ -31,6 +34,8 @@ class DepartmentInputActivity :
         initSchoolDepartmentListAdapter()
         setupNextButton()
         setupEditTextListener()
+        postDepartments()
+        observeDepartmentList()
```

Comment:
P2 : 옵저버 함수는 setup~Observe()로 해용
```suggestion
        setupDepartmentObserve()
```

Reply:
반영완!